### PR TITLE
Fix default page size for RMM data table

### PR DESCRIPTION
**Pull request by onursumer**
Related to cBioPortal/cbioportal#8062

## Checks
- [ ] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.
- [ ] The commit log is comprehensible. It follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!

### CODE REVIEW

File: packages/react-mutation-mapper/src/component/dataTable/DataTable.tsx [hunk 1]
```diff
@@ -162,17 +162,6 @@ export default class DataTable<T> extends React.Component<
...
-    @computed
-    get defaultPageSize() {
-        const initialItemsPerPage = this.props.initialItemsPerPage;
-
-        return this.tableData
-            ? this.tableData.length > initialItemsPerPage!
-                ? initialItemsPerPage
-                : this.tableData.length
-            : 1;
-    }
```
Comment (inodb) [inline packages/react-mutation-mapper/src/component/dataTable/DataTable.tsx:174]: @onursumer how come we can get rid of this logic now?

**Review by inodb — APPROVED** [review]
LGTM - minor comment: could be nice to add some clarification as to why the old logic is no longer necessary

File: packages/react-mutation-mapper/src/component/dataTable/DataTable.tsx (hunk 1 above)
Comment (onursumer) [reply packages/react-mutation-mapper/src/component/dataTable/DataTable.tsx:174]: @inodb Apparently this wasn't necessary in the first place. I thought this was required to get rid of the empty padding rows, but actually setting `minRows` to a small number is good enough. See the `minRows` property here: https://github.com/tannerlinsley/react-table/tree/v6#props.

Setting `defaultPageSize` to a computed value, especially to zero, messes things for the pagination.

**Review by leexgh — APPROVED** [review]

**Comment by onursumer** [comment]
Failing e2e tests are not related, they are also failing on master. Merging.

**Pull request merged**